apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -101,6 +101,9 @@ def get(user, query_server=None, cluster=None):
   if query_server is None:
     query_server = get_query_server_config(connector=cluster)
 
+  if not query_server.get('auth_username'):
+    query_server['auth_username'] = user.username
+
```

Comment:
@amitsrivastava can you please add a comment why we need this?

Reply:
Done